src/winetricks

```diff
@@ -14755,6 +14755,10 @@ load_uplay()
 
     # NSIS installer
     w_try "$WINE" UplayInstaller.exe ${W_OPT_UNATTENDED:+ /S}
+
+    # Get workarounds
+    w_call corefonts
+    ## TODO: winvista environment overides for upc.exe && UbisoftGameLauncher.exe
```

Comment:
You want w_set_app_winver():
https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L2632

Reply:
will finish once i'm available, busy atm